packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -168,6 +170,10 @@ export abstract class FluidDataStoreContext extends EventEmitter implements
         return this._attachState;
     }
 
+    public get isRootDataStore(): boolean {
```

Comment:
What is the scenario where we want to expose it for?
I'd rather not expose it (you would never be able to take it back), unless there is very good reason to have it.

Reply:
I exposed it in my GC draft because I was using it. But I don't think we need to. Removed exposing it.